openslides/utils/rest_api.py

```diff
@@ -46,8 +49,8 @@ def initial(self, request, *args, **kwargs):
 
         # Ensure that the incoming request is permitted
         self.perform_authentication(request)
-        if not self.check_projector_requirements():
-            self.check_permissions(request)
+        if not self.check_permissions(request) and not self.check_projector_requirements():
```

Comment:
@ostcar As you see here the projector_requirements and evaluated only if the first check returns False.